test/e2e/specs/editor/various/core-settings.spec.js

```diff
@@ -16,22 +16,33 @@ async function getOptionsValues( selector ) {
 
 // It might make sense to include a similar test in WP core (or move this one over).
 // See discussion here: https://github.com/WordPress/gutenberg/pull/32797#issuecomment-864192088.
-describe( 'Settings', () => {
-	test( 'Regression: updating a specific option will only change its value and will not corrupt others', async () => {
+test.describe( 'Settings', () => {
+	test( 'Regression: updating a specific option will only change its value and will not corrupt others', async ( {
+		page,
+		admin,
+	} ) => {
 		// We won't select the option that we updated and will also remove some
 		// _transient options that seem to change at every update.
 		const optionsInputsSelector =
 			'form#all-options table.form-table input:not([id*="_transient"]):not([id="blogdescription"])';
-		const optionsBefore = await getOptionsValues( optionsInputsSelector );
+		const optionsBefore = await getOptionsValues(
+			optionsInputsSelector,
+			admin,
+			page
+		);
 
-		await visitAdminPage( 'options-general.php' );
+		await admin.visitAdminPage( 'options-general.php' );
 		await page.type(
 			'input#blogdescription',
 			'Just another Gutenberg site'
 		);
 		await page.click( 'input#submit' );
```

Comment:
```suggestion
		await page
			.getByRole( 'textbox', { name: 'Tagline' } )
			.fill( 'Just another Gutenberg site' );
		await page.getByRole( 'button', { name: 'Save Changes' } ).click();
```

Role-based selectors should be preferred when possible. See https://playwright.dev/docs/locators#quick-guide.

Reply:
Thanks for the suggestion @Mamaduka . Old Selectors are now updated by role-based selectors as suggested.

Reply:
Thank you, @fai-sal!